etc/install_buf.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Substitute BIN for your bin directory.
+# Substitute VERSION for the current released version.
+
+if [[ -z "$1" ]]; then
+    echo "please provide an installation direction"
+    exit 1
+fi
+
+BIN="$1"
+VERSION="1.9.0"
```

Comment:
(q) I may be missing context here, but why are we on `1.9.0`? It looks like latest is now `1.16.0`, do we ever expect to update?